README.md

```diff
@@ -27,6 +27,12 @@ Docker
 The docker image is available
 [here](https://hub.docker.com/r/cgra/cgra-flow/tags).
 
+> **Recommended machine configuration**
+> * CPU: >= 6 cores
+> * Memory: 25~30G
+> 
+> **2x2 tiles run time**: ~ 40 hours (clk_period=1000)
```

Comment:
Can provide the rough hours for `clk_period=100000`? And also provide the link/location of the `clk_period` that user needs to change, i.e., `tools/OpenRoad-xxxx`.

The constraint and config files only exist in the docker but do not exist in CGRA-Flow repo. We may need to upload them and fix their path in the GUI in the next PRs.

Reply:
Sure, testing the version submodules, will fill run times for each clk period once submodules works.
Yes, will add the both config files.

Reply:
I think 1000, 10000, 100000, the 3 configurations are good enough. Thanks!